includes/class-wc-payments-utils.php

```diff
@@ -785,20 +889,12 @@ public static function cache_minimum_amount( $currency, $amount ) {
 	 * Checks if there is a minimum amount required for transactions in a given currency.
 	 *
 	 * @param string $currency The currency to check for.
-	 * @param bool   $fallback_to_local_list Whether to fallback to the local Stripe list if the cached value is not available.
 	 *
 	 * @return int|null Either the minimum amount, or `null` if not available.
 	 */
-	public static function get_cached_minimum_amount( $currency, $fallback_to_local_list = false ) {
+	public static function get_cached_minimum_amount( $currency ) {
 		$cached = get_transient( 'wcpay_minimum_amount_' . strtolower( $currency ) );
-
-		if ( (int) $cached ) {
-			return (int) $cached;
-		} elseif ( $fallback_to_local_list ) {
-			return self::get_stripe_minimum_amount( $currency );
-		}
-
-		return null;
+		return (int) $cached ? (int) $cached : null;
 	}
```

Comment:
Similar to the previous comment, reverting this back to it's state prior to #9355.